### PR TITLE
Fix installer on debian docker

### PR DESCRIPTION
**Pull request by rdettai**
Follow up to https://github.com/tenzir/tenzir/pull/3502 that fixes the failing CI: https://github.com/tenzir/tenzir/actions/runs/6230415620/job/16926081974

### CODE REVIEW

File: scripts/install.sh
Comment (mavam) [inline scripts/install.sh:178]:
```suggestion
    echo "Could not find ${bold}adduser${normal} in \$PATH."
```

**Review by mavam — APPROVED** [review]

File: scripts/install.sh
Comment (mavam) [reply scripts/install.sh:178]: It's clear if we need if we err. Given that we don't say that for other errors, I'd just remove it here.

Comment (rdettai) [reply scripts/install.sh:178]: Yeah I hesitated. It's not really obvious why `adduser` would be needed here (compared to sudo or curl). I thought adding some rational in the error would be at the same time a hint for the user and for the script maintainer.

Comment (mavam) [reply scripts/install.sh:178]: But you're not saying *why* it's needed. That would help. But it's obvious that it's needed if we bail out.

Comment (rdettai) [reply scripts/install.sh:178]: haha, I actually don't know _why_, I only know _who_ 😄

Comment (rdettai) [reply scripts/install.sh:178]: I can blame but not explain nor solve!

Comment (mavam) [reply scripts/install.sh:178]: User or developer first? Your call. 😉

Comment (rdettai) [reply scripts/install.sh]: developer relationship as a comment, user relationship to stdout!

**Pull request merged**
